deploy/manifests/nginx-gateway.yaml

```diff
@@ -87,26 +108,29 @@ spec:
       shareProcessNamespace: true
       serviceAccountName: nginx-gateway
       volumes:
-      - name: nginx-config
+      - name: nginx
         emptyDir: { }
+      - name: nginx-conf
+        configMap:
+          name: nginx-conf
       - name: var-lib-nginx
         emptyDir: { }
       - name: njs-modules
         configMap:
           name: njs-modules
       initContainers:
-      - image: busybox:1.34 # FIXME(pleshakov): use gateway container to init the Config with proper main config
-        name: nginx-config-initializer
-        command: [ 'sh', '-c', 'echo "load_module /usr/lib/nginx/modules/ngx_http_js_module.so; events {}  pid /etc/nginx/nginx.pid; error_log stderr debug; http { include /etc/nginx/conf.d/*.conf; js_import /usr/lib/nginx/modules/njs/httpmatches.js; server_names_hash_bucket_size 256; server_names_hash_max_size 1024; }" > /etc/nginx/nginx.conf && (rm -r /etc/nginx/conf.d /etc/nginx/secrets; mkdir /etc/nginx/conf.d /etc/nginx/secrets && chown 1001:0 /etc/nginx/conf.d /etc/nginx/secrets)' ]
+      - image: busybox:1.36
```

Comment:
the approach looks good to me 👍 

I just wonder if we're all ok with the busy box container from security perspective and distribution perspective, since it is a third party container -- the existing two are managed by our organization. 

and if we're ok , do we want to limits its capabilities and access to service token (that could be done separately)

Reply:
> Worth limiting capabilities at some point though for extra security.

Could you create an issue?

Reply:
https://github.com/nginxinc/nginx-kubernetes-gateway/issues/696